runtime/doc/repeat.txt

```diff
@@ -762,6 +765,29 @@ the command after changing the plugin help: >
 	:helptags path/start/foobar/doc
 	:helptags path/opt/fooextra/doc
 
+The messages that are in the lang/<lang_id>/LC_MESSAGES/foo.po file need to be
+translated to a format that the |gettext()| function understands by running the
+*msgfmt* program. This will result in a lang/<lang_id>/LC_MESSAGES/foo.mo
+file. See |multilang| on how to specify languages.
+
+In your plugin, you need to call the |bindtextdomain()| function as follows.
+This assumes that the directory structure is as above: >
+	:call bindtextdomain("foo", getcwd()~"../lang")
```

Comment:
that `getcwd()~"../lang"` looks wrong. I guess what is meant is more like this (assuming it should point to the lang directory in a directory just above where the current script is located)?
```
fnamemodify(expand("<script>"), ':p:h') .. '/../lang/')
```

Reply:
Added that, thanks for the pointer! Vim script is not my forte, I'm better at the C-side of things :-/